--- empty-1.pddl	2022-04-28 21:57:24.544260574 -0300
+++ empty-2.pddl	2022-04-29 00:41:49.465755947 -0300
@@ -117,4 +117,109 @@
 	:precondition (and )
 	:effect (and
 	)
+)
+
+(:action goto-tir-pre
+	:parameters (?oi2 - oi2 ?w - w)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-tir
+	:parameters (?oi1 - oi1 ?oi2 - oi2 ?w - w ?i - i)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-rirc1-pre
+	:parameters (?w1 - w1 ?w2 - w2)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-rirc1
+	:parameters (?oi - oi ?w1 - w1 ?w2 - w2 ?i - i ?iq - iq)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-rirc2-pre
+	:parameters (?w - w)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-rirc2
+	:parameters (?oi1 - oi1 ?oi2 - oi2 ?w - w ?iq - iq)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-cir
+	:parameters (?oi1 - oi1 ?oi2 - oi2 ?b - b)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-cir-post
+	:parameters (?b - b)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-ric1
+	:parameters (?w - w ?s - s ?sq - sq)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-ric2-pre
+	:parameters (?w1 - w1)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-ric2
+	:parameters (?oi - oi ?w1 - w1 ?w2 - w2 ?sq - sq)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-ci1
+	:parameters (?oi - oi ?w2 - w2 ?b - b)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-ci1-post
+	:parameters (?b - b)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-ci2-pre
+	:parameters (?oi2 - oi2)
+	:precondition (and )
+	:effect (and
+	)
+)
+
+(:action goto-ci2
+	:parameters (?oi1 - oi1 ?oi2 - oi2 ?w - w)
+	:precondition (and )
+	:effect (and
+	)
 ))
